app/proguard.cfg

```diff
@@ -70,27 +70,9 @@
 }
 -dontwarn com.bumptech.glide.load.resource.bitmap.VideoDecoder
 
-# for ViewPageIndicator problems (https://github.com/JakeWharton/ViewPagerIndicator/issues/366):
--dontwarn com.viewpagerindicator.LinePageIndicator
-
 # for some reason ProGuard removes this file. Why? Unsure.
 -keep class de.danoeh.antennapod.core.cast.SwitchableMediaRouteActionProvider { *; }
 
-# Retrofit 2.0
--dontwarn retrofit2.**
--keep class retrofit2.** { *; }
--keepattributes Signature
--keepattributes Exceptions
-
--keepclasseswithmembers class * {
-    @retrofit2.http.* <methods>;
-}
-
-# Moshi
--keep class com.squareup.moshi.** { *; }
--keep interface com.squareup.moshi.** { *; }
--keep public class retrofit2.adapter.rxjava.RxJavaCallAdapterFactory { *; }
```

Comment:
I think Moshi and Retrofit are used in fyydlin

Reply:
I'll change it back just in case then.